src/app/ipv/middleware.js

```diff
@@ -133,6 +133,16 @@ async function handleBackendResponse(req, res, backendResponse) {
       `/ipv/page/${req.session.currentPage}`,
     );
   }
+
+  const err = new Error(`Unknown backend response received ${backendResponse}`);
```

Comment:
Can we include the `backendResponse` as JSON? I think this will likely end up as `Unknown backend response received [object Object]`, which isn't very useful!

I think we probably either log a dedicated message for this, e.g.
```
const message = {
  description: "Unexpected backend response",
  data: backendResponse
}
req.log.error({ message, level: "ERROR" });
throw new Error(message.description);
```
or we can try including the response as a property on the error itself, and rely on the error handler to include it:
```
const err = new Error("Unexpected backend response");
err.data = backendResponse;
throw err;
```

(Either way it'd be nice to test, and see how comprehensible the log messages are 🙂)